### PR TITLE
Add `replaces` and `containerImage` to operator manifest

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
We now also update the release process to automatically adjust those versions.

#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Automatically adding `replaces` and `containerImage` to OperatorHub manifest bundle.
```

### CODE REVIEW

File: bundle/manifests/security-profiles-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -98,6 +98,7 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Security
+    containerImage: registry.k8s.io/security-profiles-operator/security-profiles-operator:v0.6.0
```
Comment (jhrozek) [inline bundle/manifests/security-profiles-operator.clusterserviceversion.yaml:101]: hmm, shouldn't this point at the `:latest` image we build in CI in between release and only point to the versioned image at the time of a release?

Comment (saschagrunert) [reply bundle/manifests/security-profiles-operator.clusterserviceversion.yaml:101]: I thought we're using the local image exclusively in the OLM tests, right?

Comment (jhrozek) [reply bundle/manifests/security-profiles-operator.clusterserviceversion.yaml:101]: Yes, we do.
Sorry, I'm confused about this change (this hunk, the `replaces` part is fine). Why are we adding the `containerImage` directive? I couldn't find much about it in OLM docs..

Comment (jhrozek) [reply bundle/manifests/security-profiles-operator.clusterserviceversion.yaml:101]: Ah, now I remember!
Thanks!

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**